components/library/libconfig/libconfig.p5m

```diff
@@ -0,0 +1,48 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 John Smith
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/include/libconfig.h
+file path=usr/include/libconfig.h++
+file path=usr/lib/cmake/libconfig++/libconfig++Config.cmake
+file path=usr/lib/cmake/libconfig/libconfigConfig.cmake
+link path=usr/lib/libconfig++.so			target=libconfig++.so.9.2.0
+link path=usr/lib/libconfig++.so.9 			target=libconfig++.so.9.2.0
+file path=usr/lib/libconfig++.so.9.2.0
+link path=usr/lib/libconfig.so 				target=libconfig.so.9.2.0
+link path=usr/lib/libconfig.so.9 			target=libconfig.so.9.2.0
+file path=usr/lib/libconfig.so.9.2.0
+file path=usr/lib/pkgconfig/libconfig++.pc
+file path=usr/lib/pkgconfig/libconfig.pc
+file path=usr/lib/$(MACH64)/cmake/libconfig++/libconfig++Config.cmake
+file path=usr/lib/$(MACH64)/cmake/libconfig/libconfigConfig.cmake
+link path=usr/lib/$(MACH64)/libconfig++.so 			target=libconfig++.so.9.2.0
+link path=usr/lib/$(MACH64)/libconfig++.so.9 		target=libconfig++.so.9.2.0
+file path=usr/lib/$(MACH64)/libconfig++.so.9.2.0
+link path=usr/lib/$(MACH64)/libconfig.so 			target=libconfig.so.9.2.0
+link path=usr/lib/$(MACH64)/libconfig.so.9 			target=libconfig.so.9.2.0
+file path=usr/lib/$(MACH64)/libconfig.so.9.2.0
+file path=usr/lib/$(MACH64)/pkgconfig/libconfig++.pc
+file path=usr/lib/$(MACH64)/pkgconfig/libconfig.pc
+file path=usr/share/info/libconfig.info 		group=bin mode=0444 owner=root \
+restart_fmri=svc:/application/texinfo-update:default 	facet.doc.info=true 
```

Comment:
fix formatting - use one space as separator and prepend restart_fmri with 4 spaces